src/compiler/checker.ts

```diff
@@ -10891,9 +10891,9 @@ module ts {
             getSymbolDisplayBuilder().buildTypeDisplay(getReturnTypeOfSignature(signature), writer, enclosingDeclaration, flags);
         }
 
-        function isUnknownIdentifier(location: Node, name: string): boolean {
+        function isUnknownIdentifier(location: Node, name: string, sourceFile: SourceFile): boolean {

```

Comment:
But `location` is used as a parameter to `resolveName` which crawls up the parent chain. How's that going to work if it has no parent?